feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -70,12 +70,13 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
             this.customerTicketingConnector
               .createTicketEvent(customerId, ticketId, ticketEvent)
               .pipe(
-                tap(() =>
-                  this.eventService.dispatch(
-                    { status: ticketEvent.toStatus?.id },
-                    TicketEventCreatedEvent
-                  )
-                )
+                tap(() => {
+                  if (ticketEvent.toStatus?.id)
+                    this.eventService.dispatch(
+                      { status: ticketEvent.toStatus?.id },
+                      TicketEventCreatedEvent
+                    );
+                })
```

Comment:
This seems that it only sends a call if a status is present.
Nevertheless, the OCC API supports adding just a message without changing the ticket status
Why are you adding now that validation?

Reply:
TicketEventCreatedEvent triggers the getTicket API and reloads the data.
I am only triggering it when the status is changed, as in other cased when message is added or attachment is uploaded, we are taking care of this from the component level.

Reason: If we reload data here, it will not wait for the upload attachment. And, if we add this to uploadAttachmentCommand, it will not be triggered when there is no attachment.
So, it's added at the component level making sure that it only triggers once.